src/main/java/spot/Main.java

```diff
@@ -0,0 +1,41 @@
+package spot;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import javafx.stage.WindowEvent;
+
+/**
+ * A GUI for Spot using FXML.
+ */
+public class Main extends Application {
+
+    private Spot spot = new Spot();
+
+    /**
+     * Starts up the Spot chatbot.
+     *
+     * @param stage Stage to display GUI nodes.
+     */
+    @Override
+    public void start(Stage stage) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
+            AnchorPane ap = fxmlLoader.load();
+            Scene scene = new Scene(ap);
+            stage.setScene(scene);
+            fxmlLoader.<MainWindow>getController().setSpot(spot);
+            stage.setOnCloseRequest((WindowEvent event) -> {
+                spot.getResponse("bye");
+            });
+            stage.setTitle("Spot");
+            stage.show();
+        } catch (IOException e) {
```

Comment:
In the start method, when catching the IOException, only the stack trace is printed. Consider adding a more descriptive error message to inform the user about the error. For example:

```java
} catch (IOException e) {
    System.err.println("Failed to load the main window.");
    e.printStackTrace();
}
```